CONTRIBUTING.md

```diff
@@ -1,20 +1,52 @@
 ## Contributing to Pyston
 
-#### Pull Requests
+### Pull Requests
 
 Before a pull request can be merged, you need to to sign the [Dropbox Contributor License Agreement](https://opensource.dropbox.com/cla/).
 
-Please make sure to run the tests (`make check` and `make check_format`) on your changes.
+Please make sure to run the tests (`make check`) on your changes.
```

Comment:
This might be a good place to mention the existence of `make quick_check` (which also checks formatting, which it didn't when we wrote this doc) and that we usually rely on travis-ci to run the full test suite anyway.